src/shared/components/SharePost.tsx

```diff
@@ -39,8 +44,19 @@ const SharePost = ({ profilePic, theme }: SharePostProps) => {
     const res = await axios.post(endPoint, postData, {
       withCredentials: true,
     });
+    const newPostInfo = {
+      ...res.data.data,
+      user: {
+        firstName: user.firstName,
+        lastName: user.lastName,
+        profilePic: user.profilePic,
+      },
+      likes: [],
+      comments: [],
+    };
     onRemove();
```

Comment:
Where are you getting this method from?

Reply:
It's a callback passed in from the Feed component.